onadata/apps/api/models/organization_profile.py

```diff
@@ -26,6 +28,19 @@ def org_profile_post_delete_callback(sender, instance, **kwargs):
     safe_delete('{}{}'.format(IS_ORG, instance.pk))
 
 
+def clear_proj_users_cache(sender, instance, created, **kwargs):
+    """
+    Signal handler to clear project users from cache.
+    """
+    if not created:
+        project_ids = cache.get(
+            f'{instance.user.username}-{OWNER_PROJECT_IDS}')
+        if project_ids:
+            for id in project_ids:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5333354)